src/session.rs

```diff
@@ -2206,12 +2206,17 @@ impl Session {
                                 }
                                 Tool::Pan(_) => {}
                                 Tool::FloodFill => {
-                                    if let Some(shapes) =
-                                        flood_fill(self.active_view(), p.into(), self.fg)
-                                    {
+                                    let start_time = time::Instant::now();
+                                    let filler =
+                                        FloodFiller::new(self.active_view(), p.into(), self.fg);
+                                    if let Some(shapes) = filler.and_then(|f| f.run()) {
                                         self.effects.push(Effect::ViewPaintFinal(shapes));
                                         self.active_view_mut().touch_layer();
                                     }
+                                    debug!(
+                                        "flood fill in: {:?} ms",
+                                        (time::Instant::now() - start_time).as_millis()
```

Comment:
I think you can just do `start_time.elapsed()`

Reply:
Oh yeah, that's much nicer.